source/process/release-metrics-definitions.rst

```diff
@@ -0,0 +1,136 @@
+|                                         | "Cannot Reproduce", "Won't fix") AND created > "START" AND created < "END"   |
++-----------------------------------------+------------------------------------------------------------------------------+
+| Valid bugs found after RC1 fixed in     | After closing current release, & adjust dates as per above:                  |
+| release                                 |                                                                              |
+|                                         | project = Mattermost AND issuetype = Bug AND resolution not in (Duplicate,   |
+|                                         | "Cannot Reproduce", "Won't Fix")  AND created > "START" AND created < "END"  |
+|                                         | AND fixVersion = latestReleasedVersion()                                     |
++-----------------------------------------+------------------------------------------------------------------------------+
+| Valid bugs found after RC1 pushed to    | After closing current release, & adjust dates as per above:                  |
+| next release                            |                                                                              |
+|                                         | project = Mattermost AND issuetype = Bug AND resolution not in (Duplicate,   |                                      
+|                                         | "Cannot Reproduce", "Won't Fix") AND created > "START" AND created < "END"   |
+|                                         | AND fixVersion = earliestUnreleasedVersion()                                 |
++-----------------------------------------+------------------------------------------------------------------------------+
+| Valid bugs found after RC1 fix version  | project = Mattermost AND issuetype = Bug AND created > "START" AND created < |  
+| = other (eg unscheduled, not set)       | "END" AND resolution not in (Duplicate, "Cannot Reproduce", "Won't Fix") AND |
+|                                         | (fixVersion not in (latestReleasedVersion(), earliestUnreleasedVersion()) OR |
+|                                         | fixVersion is EMPTY)                                                         |
++-----------------------------------------+------------------------------------------------------------------------------+
+| (Non-security) Bugs requiring patch     | After any patch release goes out (after the normal release date):            |
+| release                                 | Check Changelog for total number of non-security patch releases.             |
++-----------------------------------------+------------------------------------------------------------------------------+
+| Total features/improvements in fix      | Check number of Story tickets completed for the release.                     |
```

Comment:
Propose a more complete Jira query, like the one used for `Total valid bugs in fix version` below